### PR TITLE
Add more tests for abs and round decimal functions

### PR DESCRIPTION
**Pull request by isadikov**

### CODE REVIEW

**Comment by isadikov** [comment]
@karteekmurthys Could you review this PR? It adds a small number of tests that I thought would be good to have. I also verified that the functions work with my local set of TPC-DS.

**Comment by isadikov** [comment]
Strange, `between("c0",multiply("c1",multiply(unaryminus("c2"),abs(round("c3")))),9083)` failed in the fuzzer (https://app.circleci.com/pipelines/github/facebookincubator/velox/23530/workflows/88dad76c-4f6a-44b1-b2af-0826ddbb0314/jobs/149320) although it is related to `spark_expression_fuzzer_test`. I am going to retrigger the tests to see if it is reproducible.

File: velox/functions/prestosql/tests/DecimalArithmeticTest.cpp [hunk 1]
```diff
@@ -378,6 +378,30 @@ TEST_F(DecimalArithmeticTest, round) {
...
+           UnscaledShortDecimal::min().unscaledValue()},
+          DECIMAL(15, 0))},
+      "round(c0)",
+      {makeShortDecimalFlatVector(
+          {UnscaledShortDecimal::max().unscaledValue(),
+           UnscaledShortDecimal::min().unscaledValue()},
+          DECIMAL(15, 0))});
+
+  // Min and max long decimals.
+  testDecimalExpr<TypeKind::LONG_DECIMAL>(
```
Comment (karteekmurthys) [inline velox/functions/prestosql/tests/DecimalArithmeticTest.cpp:395]: There is min and max long decimals at L378. Do we still need this.

Comment (isadikov) [reply velox/functions/prestosql/tests/DecimalArithmeticTest.cpp:395]: The tests are for the max values for short and long decimals with no scale to check whether there might be an overflow happening depending on the calculation. I think L378 tests verify DECIMAL(38,1), these tests check DECIMAL(38,0).

Comment (karteekmurthys) [reply velox/functions/prestosql/tests/DecimalArithmeticTest.cpp:395]: Ok, would you pls mark it ready for review.

**Comment by isadikov** [comment]
@mbasmanova Could you review this PR? Thanks.

**Review by mbasmanova — COMMENTED** [review]
@bikramSingh91 Bikram, would you help review this PR?

**Review by aditi-pandit — APPROVED** [review]
Thanks Ivan for these tests.

**Comment by isadikov** [comment]
@mbasmanova Could you review this PR, it is a small test change so maybe we can merge it? Thanks.

**Review by mbasmanova — COMMENTED** [review]
@Yuhta Jimmy, would you help review this PR?

**Pull request closed without merging**